internal/store/generic.go

```diff
@@ -328,12 +329,20 @@ func (s *GenericStore[P, M, A, AL]) UpdateStatus(ctx context.Context, orgId uuid
 		return nil, err
 	}
 
-	result := s.db.WithContext(ctx).Model(&model).Where("org_id = ? AND name = ?", orgId, model.GetName()).Updates(
+	result := s.db.WithContext(ctx).Model(model).Where("org_id = ? AND name = ?", orgId, model.GetName()).Clauses(clause.Returning{}).Updates(
```

Comment:
Shouldn't the model be passed as a pointer in order for it to be updated with the results?